zwave_js_server/model/value.py

```diff
@@ -11,7 +11,7 @@ def value_id(node: "Node", event: dict) -> str:
     """Return ID of value."""
     command_class = event.get("commandClass")
     endpoint = event.get("endpoint")
```

Comment:
It's weird that we accept None value below in the f-string. Is that really intended?

Reply:
property should always be there

Reply:
If these are not optional we should use `dict[key]`.

Reply:
the value id should be:

node_id + endpoint_id + command_class_id + property + optional propertyKey

Reply:
Endpoint is optional according to the docs.

Reply:
I've also changed commandclass as it is mandatory too

Reply:
So is it ok to have a string `"None"` in the value id for the endpoint position?

Reply:
I think it's fine. We can also do `event.get('endpoint', "")` to just have empty string.

Reply:
hmm, cosmetical but maybe we should build the string instead or convert it some other presentation

Reply:
It's important the `-` are still there to wrap it.

Reply:
Leave it as is ?

Reply:
At minimum we should document the intention in a docstring or comment.